src/tasks/index.ts

```diff
@@ -528,454 +319,27 @@ export default new class TaskQueue {
    * @param {SapVatLiableEnum | null} sapVatLiable vat liable from SAP
    * @requires {String} vat liable in application format
    */
-  private translateSapVatLiable(sapVatLiable: SapVatLiableEnum | null) {
+  private translateSapVatLiable = (sapVatLiable?: SapBusinessPartner.VatLiableEnum) => {
+    if (!sapVatLiable) {
+      return null;
+    }
+
     switch (sapVatLiable) {
-      case SapVatLiableEnum.Y:
-        return "YES";
-      case SapVatLiableEnum.N:
-        return "NO";
-      case SapVatLiableEnum.EU:
+      case SapBusinessPartner.VatLiableEnum.Fi:
+        return "true";
```

Comment:
Changed this back and changed also the parsing in contacts service to be compatible with the previous values in here, since it was parsed according to "true" and "false" values.